### PR TITLE
fix: Change num_steps param to num_interpolation_steps

### PR DESCRIPTION
**Pull request by Niravprajapati1**

### CODE REVIEW

File: stable_diffusion_videos/stable_diffusion_walk.py [hunk 1]
```diff
@@ -170,7 +170,7 @@ def walk(
         data = json.load(open(prompt_config_path))
         prompts = data['prompts']
         seeds = data['seeds']
-        num_steps = data['num_steps']
+        num_interpolation_steps = data['num_interpolation_steps']
```
Comment (nateraw) [inline stable_diffusion_videos/stable_diffusion_walk.py:173]: Maybe we should have some backwards compatibility here - check first if `num_interpolation_steps` exists in prompt config. If not, check if `num_steps` is there.

File: stable_diffusion_videos/app.py [hunk 2]
```diff
@@ -60,7 +60,7 @@ def fn_videos(
         guidance_scale=guidance_scale,
         prompts=prompts,
         seeds=seeds,
-        num_steps=num_walk_steps,
+        num_interpolation_steps=num_walk_steps,
```
Comment (nateraw) [inline stable_diffusion_videos/app.py:63]: Might as well change `num_walk_steps` to `num_interpolation_steps` as well

**Review by nateraw — COMMENTED** [review]
TYSM for this contribution!! ❤️ much appreciated. Just a couple comments here. If you have any questions, please feel free to ask.

File: stable_diffusion_videos/stable_diffusion_walk.py (hunk 1 above)
Comment (Niravprajapati1) [reply stable_diffusion_videos/stable_diffusion_walk.py:173]: how can i do that ?

File: stable_diffusion_videos/app.py [hunk 3]
```diff
@@ -60,7 +60,7 @@ def fn_videos(
         guidance_scale=guidance_scale,
         prompts=prompts,
         seeds=seeds,
-        num_steps=num_walk_steps,
+        num_interpolation_steps=num_interpolation_steps,
```
Comment (nateraw) [inline stable_diffusion_videos/app.py:63]: Needs to be changed in function definition as well! After that should be good here.

**Review by nateraw — CHANGES_REQUESTED** [review]
One more quick change. Then should be ok

**Comment by Niravprajapati1** [comment]
Anything else I can do?

**Comment by nateraw** [comment]
Sorry just getting back to this now...been working on other things Will check on it today and get back to you!

**Comment by nateraw** [comment]
Off the top of my head - we will definitely need to update the README and the notebook to reflect the changes here.

I can do both, but if you get to the README before me, I would appreciate it :).

Don't bother messing with the notebook - I edit it directly in Colab.

**Comment by nateraw** [comment]
actually it got a little complicated with merge conflict - I can take it from here.

**Comment by nateraw** [comment]
Closing this and fixing in #61

**Pull request closed without merging**